<?xml version="1.0" encoding="UTF-8"?>
<!--
 * STIGQter - STIG fun with Qt
 * 
 * Copyright Â© 2018 Jon Hood, http://www.hoodsecurity.com/
 * 
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 * 
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 * 
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>STIGQter</class>
 <widget class="QMainWindow" name="STIGQter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STIGQter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabDB">
      <property name="toolTip">
       <string>Main Database</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Main DB</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="lstCCIs"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnClearCCIs">
            <property name="text">
             <string>Clear CCIs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImportCCIs">
            <property name="text">
             <string>Import CCIs</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="lstSTIGs"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnClearSTIGs">
            <property name="text">
             <string>Delete Selected STIG(s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCreateCKL">
            <property name="text">
             <string>Create CKL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImportSTIGs">
            <property name="text">
             <string>Import STIG(s)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="lstCKLs"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Delete CKL(s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImportCKL">
            <property name="text">
             <string>Import CKL(s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnOpenCKL">
            <property name="text">
             <string>Open CKL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="statusTip">
           <string/>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnFindingsReport">
        <property name="text">
         <string>Export Findings Report</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuit">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_RMF_Controls"/>
    <addaction name="actionImport_STIG_Content"/>
    <addaction name="actionClear_Database"/>
    <addaction name="separator"/>
    <addaction name="actionSave_CKL"/>
    <addaction name="action_Open_CKL"/>
    <addaction name="separator"/>
    <addaction name="action_Help"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_RMF_Controls">
   <property name="text">
    <string>Import &amp;RMF Controls</string>
   </property>
  </action>
  <action name="actionImport_STIG_Content">
   <property name="text">
    <string>Import S&amp;TIG Content</string>
   </property>
  </action>
  <action name="actionClear_Database">
   <property name="text">
    <string>&amp;Clear Database</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSave_CKL">
   <property name="text">
    <string>&amp;Save CKL</string>
   </property>
  </action>
  <action name="action_Open_CKL">
   <property name="text">
    <string>&amp;Open CKL</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnQuit</sender>
   <signal>pressed()</signal>
   <receiver>STIGQter</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>STIGQter</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImportCCIs</sender>
   <signal>clicked()</signal>
   <receiver>STIGQter</receiver>
   <slot>UpdateCCIs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UpdateCCIs()</slot>
 </slots>
</ui>
